types/jquery.pjax/tslint.json

```diff
@@ -74,6 +74,7 @@
         "typedef-whitespace": false,
         "unified-signatures": false,
         "void-return": false,
-        "whitespace": false
+        "whitespace": false,
+        "no-angle-bracket-type-assertion": false
```

Comment:
Why disable this TSLint rule? I suppose we could better to enable it. (What is the default value?)

Reply:
Those rule failures were caused by an unrelated change (https://github.com/Microsoft/dtslint/pull/141).